### PR TITLE
Complete API refactor for chosen database

### PR DESCRIPTION
**Pull request by anabellchan**

### CODE REVIEW

File: controller/products.js [hunk 1]
```diff
@@ -24,34 +25,27 @@ const sellers = (req, res) => {
...
-    })
-    .then((productData) => {
-      priceInfo = productData;
-      return true;
-    })
-    .then(() => createQuotes(priceInfo, sellerInfo, req.params.sellerLimit))
-    .then((quoteData) => {
-      if (!quoteData.length) {
-        return res.status(404).send('Product Not Found.');
+  retrieveSellers(id)
```
Comment (jamesstockman14) [inline controller/products.js:36]: This refactor is a lot easier to read, nice work.

File: services/mongodb/quotes.js [hunk 2]
```diff
@@ -53,3 +54,14 @@ module.exports.createQuotes = (prices, sellers, limit) => {
...
+
+let priceInfo;
+let sellerInfo;
+module.exports.createQuotes = (data, id) => {
+  sellerInfo = data;
+  return retrievePrices(id)
+    .then((productData) => {
+      priceInfo = productData;
+      return sellerData(priceInfo, sellerInfo, 10);
+    });
```
Comment (jamesstockman14) [inline services/mongodb/quotes.js]: should this have a catch?

File: services/mongodb/quotes.js [hunk 3]
```diff
@@ -53,3 +54,14 @@ module.exports.createQuotes = (prices, sellers, limit) => {
...
 };
+
+let priceInfo;
+let sellerInfo;
+module.exports.createQuotes = (data, id) => {
+  sellerInfo = data;
+  return retrievePrices(id)
+    .then((productData) => {
+      priceInfo = productData;
+      return sellerData(priceInfo, sellerInfo, 10);
```
Comment (jamesstockman14) [inline services/mongodb/quotes.js:65]: should this have a catch?

**Review by jamesstockman14 — COMMENTED** [review]
Looks good :)

**Comment by anabellchan** [comment]
Nice catch @jamesstockman14!  Fixed it.

**Pull request merged**